framework/topology.cpp

```diff
@@ -656,19 +534,65 @@ static bool fill_ppin_sysfs(struct cpu_info *info)
     return false;
 }
 
+static unsigned  get_apic_id_from_cpuid() {
+    #ifdef _WIN32
+        unsigned int cpu_info[4];
+        __cpuid(cpu_info, 1);
+        return (cpu_info[1] >> 24) & 0xFF;
```

Comment:
Doesn't make sense. `__cpuid` comes from GCC's cpuid.h header, which defines it as:
```c
#define __cpuid(level, a, b, c, d)					\
  __asm__ __volatile__ ("cpuid\n\t"					\
			: "=a" (a), "=b" (b), "=c" (c), "=d" (d)	\
			: "0" (level))
```